### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -462,7 +462,7 @@ npm install gmsmith
 Alternatively, we can use `--save` or `--save-dev` to save to our `package.json's dependencies` or `devDependenices`.
 
 ```bash
-npm install gmsmith --save  # Updates {"dependencies": {"gmsmith": "1.2.3"}}
+npm install gmsmith --save      # Updates {"dependencies": {"gmsmith": "1.2.3"}}
```
Comment (twolfson) [inline README.md]: This repo prefers to not align comments, please revert this change

**Comment by twolfson** [comment]
Awesome, thanks for the touchup =) There is one spot that I would like to undo (see inline comment). Can you please update it?

**Comment by XhmikosR** [comment]
Done.

**Comment by twolfson** [comment]
Cool, looks good. This has been landed/released in `4.6.1`. Thanks again for the PR.
